src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -72,7 +72,8 @@ class CodeActionOracle extends Disposable {
 		if (selection.isEmpty() && trigger.type === CodeActionTriggerType.Auto) {
 			const { lineNumber, column } = selection.getPosition();
 			const line = model.getLineContent(lineNumber);
-			if (line.length > 0) {
+			if (!/^\s*$/g.test(line)) {
```

Comment:
I think that check is ok. We just need to modify the behavior when we see that we are at the end of a line that is all whitespace

Reply:
So this check is related to the comment I posted here https://github.com/microsoft/vscode-copilot/pull/3165#discussion_r1411909632 and is the counterpart of the logic on the VS Code side.